write-fonts/src/graph.rs

```diff
@@ -203,17 +203,62 @@ impl Graph {
         }
     }
 
+    /// Attempt to pack the graph.
+    ///
+    /// This involves finding an order for objects such that all offsets are
+    /// resolveable.
+    ///
+    /// In the simple case, this just means finding a topological ordering.
+    /// In exceptional cases, however, this may require us to significantly
+    /// modify the graph.
+    ///
     /// returns `true` if a solution is found, `false` otherwise
-    pub(crate) fn topological_sort(&mut self) -> bool {
+    pub(crate) fn pack_objects(&mut self) -> bool {
```

Comment:
this should reference prior and relevant art. I believe in this case that means harfbuzz and in particular Garret's writing on the topic?